Buildings/Controls/OBC/CDL/Discrete/DiscreteMovingMean.mo

```diff
@@ -0,0 +1,104 @@
+within Buildings.Controls.OBC.CDL.Discrete;
+block DiscreteMovingMean "Discrete moving mean of a sampled input signal"
+  parameter Integer nSam(min=2) "Number of samples to be averaged";
+  parameter Modelica.SIunits.Time samplePeriod(min=1E-3)
+      "Sampling period of component";
+  parameter Real y_start = 0
+      "Initial value for the first nSam-1 samples";
+  parameter Modelica.SIunits.Time startTime = 0
```

Comment:
This should be removed. See `FirstOrderHold` for how we deal with the simulation start time.